src/python/ensembl/io/genomio/genome_stats/compare.py

```diff
@@ -189,11 +191,32 @@ def main() -> None:
     )
     parser.add_argument_src_path("--ncbi_stats", required=True, help="NCBI dataset JSON file")
     parser.add_argument_src_path("--core_stats", required=True, help="Core database JSON file")
+    parser.add_argument("-v", "--verbose", action="store_true", required=False,
+        help="Verbose level logging")
+    parser.add_argument("-d", "--debug", action="store_true", required=False,
+        help="Debug level logging")
     args = parser.parse_args()
 
+    # Configure logging
+    date_format='%Y/%m/%d_%I:%M:%S(%p)'
+    logging_format='%(asctime)s - %(levelname)s - %(message)s'
+    reload(logging)
```

Comment:
I'm still trying to figure out why this is needed (and, hence, if it is needed in my PR). I've seen this in @Dishalodha's PR as well, so I have removed the "if" and left only the "why" question.